testing/events.go

```diff
@@ -129,3 +130,33 @@ func ParseAckFromEvents(events sdk.Events) ([]byte, error) {
 	}
 	return nil, fmt.Errorf("acknowledgement event attribute not found")
 }
+
+// AssertEvents asserts that expected events are present in the actual events
+// expected map needs to be a subset of actual events to pass
+func AssertEvents(
+	suite *suite.Suite,
+	expected map[string]map[string]string,
```

Comment:
is there a reason to use `map[string]map[string]string` as opposed to taking an `sdk.Events` directly?

Reply:
Just wanted the map of expected events being readable as a map for the test itself.

We can change this to `sdk.Events` as well, but i chose readability of expected events. Do you think we should change it?

Reply:
Cool, this snipet is the reason, to me this just makes intutive sense of what to expect from the event. https://github.com/cosmos/ibc-go/pull/2829/files#diff-ae66bd29b0c68b16e3fc64fb2d8c88dcc90ef4ed6c1f67be36baf8d693473f8eR150-R156

Reply:
Would it be possible though to make a type definition for `map[string]map[string]string` just to improve readability? Like `eventsMap` or something like that? Something that signals the purpose of the type more clearly than the mouthful of `map[string]map[string]string`. :)

Reply:
makes sense will do